src/key.rs

```diff
@@ -148,6 +148,20 @@ impl SecretKey {
         }
     }
 
+    #[inline]
```

Comment:
Minor nit - we probably shouldn't use `#[inline]`, I think these days we can trust the Rust compiler to just DTRT here